ursula_cli/shell.py

```diff
@@ -331,7 +331,7 @@ def _run_heat(args, hot):
 """
 
     # write out private key if using one generated by heat
-    if private_key != "":
+    if private_key != "" and private_key is not None:
```

Comment:
Yeah, I guess I can't think of any situation where a valid private key
would evaluate to False! Changing now.

On Tue, Aug 2, 2016 at 5:00 PM, Jesse Keating <notifications@github.com>
wrote:

> In ursula_cli/shell.py
> <https://github.com/blueboxgroup/ursula-cli/pull/53#discussion_r73184651>:
>
> > @@ -331,7 +331,7 @@ def _run_heat(args, hot):
> >  """
> >
> >      # write out private key if using one generated by heat
> > -    if private_key != "":
> > +    if private_key != "" and private_key is not None:
>
> Why not just if not private_key:
>
> Python treats "" and None as boolean False.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/blueboxgroup/ursula-cli/pull/53/files/c20ab97b8894a8592ff24d2d6714b4c1a91490ef#r73184651>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADKSsqSVuH67j5C_Kkwuc75dgWHwhHA7ks5qb2mbgaJpZM4JajNM>
> .
>